service/src/main/kotlin/fi/espoo/evaka/serviceneednew/NewServiceNeedController.kt

```diff
@@ -104,7 +109,9 @@ class NewServiceNeedController(
         acl.getRolesForNewServiceNeed(user, id).requireOneOfRoles(UserRole.ADMIN, UserRole.UNIT_SUPERVISOR)
 
         db.transaction { tx ->
+            val childRange = tx.getNewServiceNeedChildRange(id)
```

Comment:
The delete query could just return this data instead.

Reply:
But then I need to somehow join to placement within delete query to get the childId?